apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -115,16 +115,24 @@ function DescriptionSection(props: DescriptionProps) {
   );
 }
 
-type CtaProps = Pick<Twc2SupportDetail, 'ctaBlurb' | 'ctaButtonText'>;
+type CtaProps = Pick<
+  Twc2SupportDetail,
+  'ctaBlurb' | 'ctaButtonText' | 'ctaLink' | 'whatsappLogoSrc'
+>;
 
 function CtaSection(props: CtaProps) {
-  const { ctaBlurb, ctaButtonText } = props;
+  const { ctaBlurb, ctaButtonText, ctaLink, whatsappLogoSrc } = props;
   return (
     <Box className="support-cta-container" sx={{ borderTopColor: 'grey.200' }}>
       <div style={{ flex: 4, height: '100%' }}>{ctaBlurb}</div>
       <PrimaryButton
+        href={ctaLink}
+        target="_blank"
+        rel="noopener"
         sx={{ flex: 5, backgroundColor: 'primary.700', height: '100%' }}
       >
+        <img src={whatsappLogoSrc} />
```

Comment:
@stevahnes I meant that it's okay to hardcode the value here. Also, I would appreciate it if you wait for my clarification before merging the PR when I set specific guidelines around merging it.

Reply:
Yeah, well you said I can merge it and work on it on my PR, so that's what I am doing @glencbz. Also, it was my suggestion to not hardcode the value because if you see the other mocks, we are using other icons for other CTA buttons.

Reply:
Sorry, I didn't mean to come across as overriding your suggestion.

What I mean is that if we have a component called `WhatsappIcon`, it doesn't make sense to parameterise the URL for the WhatsApp image (especially if it's also called whatsappLogoSrc) because we're only ever going to have one anyway.

If we want a generic CTA that can accept any image URL, then sure, parameterising is fine. Either approach can work as long as it's coherent.